pkg/controller/command/models.go

```diff
@@ -248,3 +250,33 @@ type UnwrapKeyRequest struct {
 type UnwrapKeyResponse struct {
 	Key []byte `json:"key"`
 }
+
+// BlindRequest serializable Blind request.
```

Comment:
These comments should form proper sentence (i.e. `BlindRequest is a serializable Blind request.`)